Missing import when running integration tests

```
    import json
    import re
    from contextlib import contextmanager
    from dataclasses import dataclass
    from functools import lru_cache
    import agate
    from requests.exceptions import ConnectionError
    from typing import Optional, Any, Dict, Tuple
    
    import google.auth
    import google.auth.exceptions
    import google.cloud.bigquery
    import google.cloud.exceptions
    from google.api_core import retry, client_info
    from google.auth import impersonated_credentials
    from google.oauth2 import (
        credentials as GoogleCredentials,
        service_account as GoogleServiceAccountCredentials
    )
    
    from dbt.utils import format_bytes, format_rows_number
>   from dbt.clients import agate_helper, gcloud
E   ImportError: cannot import name 'gcloud' from 'dbt.clients' (/Users/kwigley/workspace/dbt-bigquery/.tox/py38-bigquery/lib/python3.8/site-packages/dbt/clients/__init__.py)
```

--- a/dbt/adapters/bigquery/connections.py
+++ b/dbt/adapters/bigquery/connections.py
@@ -18,8 +18,9 @@ from google.oauth2 import (
     service_account as GoogleServiceAccountCredentials
 )
 
+from dbt.adapters.bigquery import gcloud
 from dbt.utils import format_bytes, format_rows_number
-from dbt.clients import agate_helper, gcloud
+from dbt.clients import agate_helper
 from dbt.tracking import active_user
 from dbt.contracts.connection import ConnectionState, AdapterResponse
 from dbt.exceptions import (

--- a/dbt/adapters/bigquery/gcloud.py
+++ b/dbt/adapters/bigquery/gcloud.py
@@ -1,0 +1,26 @@
+from dbt.logger import GLOBAL_LOGGER as logger
+import dbt.exceptions
+from dbt.clients.system import run_cmd
+
+NOT_INSTALLED_MSG = """
+dbt requires the gcloud SDK to be installed to authenticate with BigQuery.
+Please download and install the SDK, or use a Service Account instead.
+
+https://cloud.google.com/sdk/
+"""
+
+
+def gcloud_installed():
+    try:
+        run_cmd('.', ['gcloud', '--version'])
+        return True
+    except OSError as e:
+        logger.debug(e)
+        return False
+
+
+def setup_default_credentials():
+    if gcloud_installed():
+        run_cmd('.', ["gcloud", "auth", "application-default", "login"])
+    else:
+        raise dbt.exceptions.RuntimeException(NOT_INSTALLED_MSG)

--- a/dbt/adapters/bigquery/impl.py
+++ b/dbt/adapters/bigquery/impl.py
@@ -4,7 +4,6 @@ from dbt.dataclass_schema import dbtClassMixin, ValidationError
 
 import dbt.deprecations
 import dbt.exceptions
-import dbt.clients.gcloud
 import dbt.clients.agate_helper
 
 from dbt import ui

